index.html

```diff
@@ -20,11 +20,52 @@
 
 
 <body>
+    <nav>
```

Comment:
- [ ] Kindly make sure to use the same color for the top nav bar as the provided design, also consider aligning the nav links in the white nav bar in the center so they are more appealing and look like the provided design

|Your page|Provided design|
|-|-|
|![Screenshot_20230113_182510](https://user-images.githubusercontent.com/25227022/212382630-75229a4b-5de4-4bfb-b046-df4e3be7ec23.png)|![Screenshot_20230113_182457](https://user-images.githubusercontent.com/25227022/212382661-2c6b6866-6bdb-4ff2-9152-76c520f1b2eb.png)|